### DISCUSSION TITLE
Read Only Mode

### DISCUSSION THREAD

**Discussion by USAFPride** [rtl-airband/RTLSDR-Airband#209] [Q&A]
Is it possible to run RTLSDR in read-only mode?  I'm pretty sure that nothing is written to the SD card during operation, but wanted to check.

**Comment by szpajder** [comment]
Moved to Discussions. Please create issues only for reporting bugs in the program.

RTLSDR-Airband writes the following data:

- audio files - if any outputs of type `file` or `rawfile` are configured
- statistics - if enabled with `stats_filepath` option
- PID file - when running in background (ie. when neither `-f` nor `-F` option is used)
- debug log - when compiled with `-DDEBUG` option

By default, the PID location is `/run/rtl_airband.pid` (it can be changed with `pidfile` configuration option). On most systems, `/run` is on a RAM disk (tmpfs) though.

Default debug file location is `rtl_airband_debug.log` in the current directory. It can be changed with `-d` command line option.

The program also produces log messages, which, by default, are sent to syslog. Depending on the syslog daemon configuration, they may get written to files (e.g. `/var/log/messages`). If you don't want that, use `-e` option to direct the logs to stderr.

In short: yes, it is possible to run the program in read only mode. Just make sure that your configuration does not satisfy any of the above conditions.
